### ISSUE TITLE
Slider - no node selected error.

### ISSUE THREAD

**Issue by rudybanks** [navelpluisje/figma-slider-creator#1]
Hi I am testing the plugin. I have my design setup properly but I keep getting a "no valid node selected for creating a knob"? I'm doing vertical 5 steps bottom-top. **Update:** I looked at the logic and noticed the plugin looked for Frame or group. I decided to Group all the components and then run the plugin and it started working.
![image](https://github.com/user-attachments/assets/c3ac18e0-d22a-4492-a1ae-1b4c221e1923)

**Comment by navelpluisje** [comment]
Hi Rudy, 
Here's a screenshot of the setup of a slider and the element to select to make it all work:
<img width="627" alt="image" src="https://github.com/user-attachments/assets/806a2abd-8027-4539-8cb1-4dff34897ca6">

Please let me know if this was helpfull.
regards,
Erwin

**Comment by rudybanks** [comment]
Hi Erwin,

Thanks for the reply.
After looking at the logic I noticed you were looking for a Frame or Group.
When I had the 4 elements in a frame (no group i.e circled in red) I got
the node error.
But after trial and error I grouped the elements and it worked.

I had another friend who struggled with this but I was able to share my
learnings.

Great work on this. I am learning Figma scripting myself.

Regards,
Rudy

[image: image.png]
